nifi-frontend/src/main/frontend/apps/nifi-jolt-transform-ui/src/app/app.module.ts

```diff
@@ -30,6 +34,14 @@ import { MAT_FORM_FIELD_DEFAULT_OPTIONS } from '@angular/material/form-field';
 import { EffectsModule } from '@ngrx/effects';
 import { JoltTransformJsonUiEffects } from './pages/jolt-transform-json-ui/state/jolt-transform-json-ui/jolt-transform-json-ui.effects';
 
+const entry = localStorage.getItem('disable-animations');
+let disableAnimations: string = entry !== null ? JSON.parse(entry).item : '';
+
+// honor OS settings if user has not explicitly disabled animations for the application
+if (disableAnimations !== 'true' && disableAnimations !== 'false') {
+    disableAnimations = window.matchMedia('(prefers-reduced-motion: reduce)').matches.toString();
```

Comment:
Same question here about whether we should protect the call to `matchMedia`.

Reply:
I think we can get rid of it in the navigation component.